data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+        "species": "annihilape",
+        "weight": 15,
+        "moves": [
+          ["uturn"],
+          ["closecombat"],
+          ["shadowclaw", "ragefist"],
+          ["finalgambit"]
+        ],
+        "item": ["choicescarf"],
+        "nature": "jolly",
+        "evs": [252, 4, 0, 0, 0, 252],
+        "teratype": ["ghost", "normal"],
+        "ability": "defiant"
+      }
+    ]
+  },
+  "heatran": {
+    "weight": 7,
+    "sets": [
+      {
+        "species": "heatran",
+        "weight": 35,
+        "moves": [
+          ["magmastorm"],
+          ["taunt"],
+          ["protect"],
+          ["substitute", "earthpower", "terablast"]
+        ],
+        "item": ["leftovers"],
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "calm",
+        "evs": [252, 0, 0, 0, 252, 4],
+        "teratype": ["grass"],
+        "wantsTera": true,
+        "ability": "flashfire"
+      },
+      {
+        "species": "heatran",
+        "weight": 40,
+        "moves": [
+          ["magmastorm"],
+          ["earthpower"],
+          ["flashcannon"],
+          ["terablast"]
+        ],
+        "item": ["assaultvest"],
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "modest",
+        "evs": [252, 0, 0, 252, 0, 4],
+        "teratype": ["grass", "fairy"],
+        "wantsTera": true,
+        "ability": "flashfire"
+      },
+      {
+        "species": "heatran",
+        "weight": 25,
+        "moves": [
+          ["magmastorm"],
+          ["flashcannon"],
+          ["terablast", "stealthrock"],
+          ["willowisp"]
+        ],
+        "item": ["sitrusberry", "rockyhelmet"],
+        "nature": "bold",
+        "evs": [252, 0, 252, 4, 0, 0],
+        "teratype": ["fairy", "grass"],
+        "wantsTera": true,
+        "ability": "flamebody"
+      }
+    ]
+  },
+  "breloom": {
+    "weight": 7,
+    "sets": [
+      {
+        "species": "breloom",
+        "weight": 80,
+        "moves": [
+          ["bulletseed"],
+          ["machpunch"],
+          ["bulldoze", "rocktomb"],
+          ["spore"]
+        ],
+        "item": ["focussash"],
+        "nature": "adamant",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["grass", "fighting", "ground"],
+        "ability": "technician"
+      },
+      {
+        "species": "breloom",
+        "weight": 10,
+        "moves": [
+          ["bulletseed"],
+          ["machpunch"],
+          ["terablast"],
+          ["spore", "swordsdance"]
+        ],
+        "item": ["loadeddice"],
+        "nature": "adamant",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["fire"],
+        "wantsTera": true,
+        "ability": "technician"
+      },
+      {
+        "species": "breloom",
+        "weight": 10,
+        "moves": [
+          ["substitute"],
+          ["leechseed", "toxic"],
+          ["spore"],
+          ["protect"]
+        ],
+        "item": ["toxicorb"],
+        "nature": "impish",
+        "evs": [204, 4, 252, 0, 28, 20],
```

Comment:
-This breloom has no attacks, and therefore does not need attack EVs, IVs, or nature. Amend this.
-20 speed doesn't do anything, Honchkrow and Sandaconda don't exist. 204/0/252/0/48/4 would be acceptable.